students/gotem888/hw-12 searchEngine/index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <meta charset = "utf-8">
+        <link rel="stylesheet" href="./css/reset.css">
+        <link rel="stylesheet" href="./css/style.css">
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300&family=Roboto:wght@300&display=swap" rel="stylesheet">
+        <title>HW - SearchEngine</title>
+    </head>
+    <body>
+        <div class="wrapper">
+            <aside class="wrapper__filters">
+                <div class="posCalibrate">
+                <form name="filtersForm" class="form" >
+                    <div class="form__item inputArea">
+                        <input type="text" required name="vstname" id="" autocomplete="off"/>
+                        <label for="">Enter VST name <p id="here">here</p></label>
+                    </div>
+                
+                <div class="radio">
+                    <div class="form__item form__item--radio">
+                        <input type="radio" name="group" id="group_1" value="instruments"/>
+                        <label for="group_1">VST Instruments</label>
+                    </div>
+
+                    <div class="form__item form__item--radio">
+                        <input type="radio" name="group" id="group_2" value="effects"/>
+                        <label for="group_2">VST Effects</label>
+                    </div>
+                </div>
+
+                    <button type="submit" class="form__item">Search</button>
+                    <button type="reset" class="form__item" id="rstButton">Reset</button>
```

Comment:
Try to give meaningful names
```suggestion
                    <button type="reset" class="form__item" id="resetButton">Reset</button>
```